### DISCUSSION TITLE
Files.app

### DISCUSSION THREAD

**Discussion by foxly-it** [blinksh/blink#1424] [Support]
Maybe I'm just being silly. I have created a folder in my iCloud Drive. In this folder I created via Blink (touch test.md). Now I want to open it in VS Code. Unfortunately this does not work. At least I haven't figured it out yet.

Now I thought ok, I try the whole thing via a host. I have already saved the location, I can open the file but if I change something and want to save the change, the saving process does not work.

Am I just being silly, or does the integration of VS Code not quite work yet.

**Comment by yury** [comment]
You need to install blink-fs extension inside vscode.

1. `code .`
2. vscode will open
3. go to extensions and search for `blink-fs` and install it.
4. Folder will shows after installation

**Reply by foxly-it** [reply]
Thanks exactly that was the problem ;-)

**Comment by foxly-it** [comment]
I am a bit confused right now, I have added a host and for this a location to Files.app. In the Files.app of iOS everything is also recognized correctly. But when I try to edit e.g. a bash script via code, I get the following error in the blinkCode.log:

```
[info] Mar 03 2022, 12:55:25 FileSystem Registered mount 12 for xxx at blinksftp://cloud/~/mnt/HETZNER/data/scripte/backup.sh
[info] Mar 03 2022, 12:55:28 CodeWebSocketServer Connection state update - preparing
[info] Mar 03 2022, 12:55:28 CodeWebSocketServer Connection state update - ready
[debug] Mar 03 2022, 12:55:28 Optional("cloud") stat /~/mnt/HETZNER/data/scripte/backup.sh
[error] Mar 03 2022, 12:55:28 Connection ::1.49428 Error completing operation - fileNotFound(uri: BlinkCode.URI(rootPath: BlinkCode.RootPath(url: blinksftp://cloud/~/mnt/HETZNER/data/scripte/backup.sh)))
[info] Mar 03 2022, 12:55:32 FileSystem De-registering mount 12
```

I have to say, before it opens VS Code, there is a popup from Blink which disappears so fast, that I can't see what information it contains.

**Reply by foxly-it** [reply]
Ok I think it is a mistake. If I open the path Manuel via CLI it works. If I open it via the location in files.App it does not work.
